libs/agno/agno/team/team.py

```diff
@@ -1007,9 +1019,12 @@ def run(
         # Update session state from DB
         session_state = self.update_session_state(session=team_session, session_state=session_state)
 
-        # Resolve dependencies
-        if self.dependencies is not None:
-            self._resolve_run_dependencies()
+        # Determine runtime dependencies
```

Comment:
Your not resolving the `add_history_to_context` or `add_dependencies_to_context` like you do for agent